imap/quota.c

```diff
@@ -387,7 +390,7 @@ int buildquotalist(char *domain, char **roots, int nroots, int isuser)
 
     /* basic case - everything (potentially limited by domain still) */
     if (!nroots) {
-        r = quota_foreach(buf, fixquota_addroot, buf, NULL);
+        r = quota_foreach(buf, fixquota_addroot, NULL, NULL);
```

Comment:
Hmm.  Does this changed line mean `quota -d domain` is no longer limited by domain?

We only have one test that exercises the `-d` option, and it doesn't check that other domains aren't affected.

Reply:
Oh no, the first argument is still "buf". The changed argument is just the fixquota_addroot rock, which was previously unused and is now a userid.  Confused myself!